ansible/roles/runner/tasks/main.yml

```diff
@@ -1,209 +1,7 @@
-  # systemd unit modified/created.
-  register: runner_projects
-  with_items: "{{ runners }}"
-  notify:
-    - reload systemd daemon
-    - restart actions-runners
-
-- name: Enable runner service
-  become: yes
-  ansible.builtin.service:
-    name: actions-runner-{{ item.0.normalized }}@{{ item.1 }}
-    enabled: yes
-    state: started
-  loop: "{{ runners | subelements('workers') }}"
-
-
-- name: Set docker pruning service
-  become: yes
-  ansible.builtin.copy:
-    dest: /etc/systemd/system/docker_pruning.service
-    content: |
-      [Unit]
-      Description=Prune unused docker resources
-      Wants=docker_pruning.timer
-
-      [Service]
-      Type=oneshot
-      ExecStart=/usr/bin/docker system prune --volumes -f
-    mode: 0644
-    owner: root
-    group: root
-  notify:
-    - reload systemd daemon
-    - restart docker_pruning timer
-
-
-- name: Set docker pruning timer
-  become: yes
-  ansible.builtin.copy:
-    dest: /etc/systemd/system/docker_pruning.timer
-    content: |
-      [Unit]
-      Description=Run docker_pruning service daily
-
-      [Timer]
-      # Run at 8:00 UTC daily
-      OnCalendar=*-*-* 08:00:00
-      [Install]
-      WantedBy=timers.target
-    mode: 0644
-    owner: root
-    group: root
-  notify:
-    - reload systemd daemon
-    - restart docker_pruning timer
-
-- name: Enable docker pruning timer
-  become: yes
-  ansible.builtin.service:
-    name: docker_pruning.timer
-    enabled: yes
-    state: started
-
-# Watchdog
-- name: Actions runner watchdog script
-  become: yes
-  ansible.builtin.copy:
-    src: actions-runner-watchdog.sh
-    dest: /usr/local/sbin/actions-runner-watchdog.sh
-    mode: 0700
-    owner: root
-    group: root
-
-- name: Set actions runner watchdog service
-  become: yes
-  ansible.builtin.copy:
-    dest: /etc/systemd/system/actions-runner-watchdog.service
-    content: |
-      [Unit]
-      Description=Kill unhealty actions runner containers
-      Wants=actions-runner-watchdog.timer
-
-      [Service]
-      Type=oneshot
-      ExecStart=/usr/local/sbin/actions-runner-watchdog.sh
-    mode: 0644
-    owner: root
-    group: root
-  notify:
-    - reload systemd daemon
-    - restart actions-runner-watchdog timer
-
-- name: Set actions runner watchdog timer
-  become: yes
-  ansible.builtin.copy:
-    dest: /etc/systemd/system/actions-runner-watchdog.timer
-    content: |
-      [Unit]
-      Description=Run actions-runner-watchdog service regularly
-
-      [Timer]
-      # Run two minutes after previous run done
-      OnBootSec=1m
-      OnUnitInactiveSec=2m
-      [Install]
-      WantedBy=timers.target
-    mode: 0644
-    owner: root
-    group: root
-  notify:
-    - reload systemd daemon
-    - restart actions-runner-watchdog timer
-
-- name: Enable actions runner watchdog timer
-  become: yes
-  ansible.builtin.service:
-    name: actions-runner-watchdog.timer
-    enabled: yes
-    state: started
+- include_tasks: main-generic.yml
```

Comment:
For my understanding, the non-existence of this file is fine because we don't use Ansible for anything but the `s390x` architecture currently, is that correct?

Reply:
correct and good catch. I did add it later in a stack to come..... https://github.com/chantra/ci/commit/f2a1c2d96511958b463c1299f55f780bca9dda96

I wanted to get this base sorted before building on top of it (and was actively iterating on it).
The current final aws + s390x work is https://github.com/chantra/ci/commits/ansible_aws but I am working on a separate stack to have better indiviual commits.